packages/functional-tests/tests/syncV3/signUp.spec.ts

```diff
@@ -0,0 +1,148 @@
+    email = login.createEmail('sync{id}');
+    await login.clearCache();
+  });
+
+  test('sign up', async ({
+    target,
+    page,
+    pages: { login, signinTokenCode, connectAnotherDevice },
+  }) => {
+    await page.goto(
+      `${target.contentServerUrl}?context=fx_desktop_v3&service=sync&action=email`,
+      { waitUntil: 'networkidle' }
+    );
+    await login.setEmail(email);
+    await signinTokenCode.clickSubmitButton();
+
+    // Verify the email is correct
+    expect(await login.getPrefilledEmail()).toMatch(email);
+
+    // Passwords do not match should cause an error
+    await login.setPassword(password);
+    await login.confirmPassword(incorrectPassword);
+    await login.setAge('21');
+    await signinTokenCode.clickSubmitButton();
+
+    // Verify the error message
+    expect(await login.getTooltipError()).toMatch('Passwords do not match');
+
+    // Fix the error
+    await login.confirmPassword(password);
+    await login.submit();
+    await login.fillOutSignUpCode(email);
+    expect(await connectAnotherDevice.fxaConnected.isVisible()).toBeTruthy();
+  });
+
+  test('coppa disabled', async ({
+    target,
+    page,
+    pages: { login, connectAnotherDevice },
+  }) => {
+    const query = { coppa: 'false' };
+    const queryParam = new URLSearchParams(query);
+    await page.goto(
+      `${
+        target.contentServerUrl
+      }?context=fx_desktop_v3&service=sync&action=email&${queryParam.toString()}`,
+      { waitUntil: 'networkidle' }
+    );
+    await login.setEmail(email);
+    await login.submit();
+    await login.setPassword(password);
+    await login.confirmPassword(password);
+
+    // Age textbox is not on the page and click submit
+    await login.submit();
+    await login.fillOutSignUpCode(email);
+    expect(await connectAnotherDevice.fxaConnected.isVisible()).toBeTruthy();
+  });
+
+  test('email specified by relier, invalid', async ({
+    target,
+    page,
+    pages: { login },
+  }) => {
+    const invalidEmail = 'invalid@@';
+    const query = { email: invalidEmail };
+    const queryParam = new URLSearchParams(query);
+    await page.goto(
+      `${
+        target.contentServerUrl
+      }?context=fx_desktop_v3&service=sync&action=email&${queryParam.toString()}`
+    );
+    expect(await login.getTooltipError()).toMatch('Valid email required');
+  });
+
+  test('email specified by relier, empty string', async ({
+    target,
+    page,
+    pages: { login },
+  }) => {
+    const emptyEmail = ' ';
+    const query = { email: emptyEmail };
+    const queryParam = new URLSearchParams(query);
+    await page.goto(
+      `${
+        target.contentServerUrl
+      }?context=fx_desktop_v3&service=sync&action=email&${queryParam.toString()}`
+    );
+    expect(await login.getTooltipError()).toMatch('Valid email required');
+  });
+
+  test('email specified by relier, not registered', async ({
+    target,
+    page,
+    pages: { login },
+  }) => {
+    const query = { email: email };
```

Comment:
nit: you can also use shorthand syntax if that looks any better (here and L134 below):
```suggestion
    const query = { email };
```